apps/qubit/modules/informationobject/actions/fullWidthTreeViewMoveAction.class.php

```diff
@@ -63,6 +63,11 @@ public function execute($request)
             return $this->renderText(json_encode(['error' => $i18n->__('Move not needed: new and old positions are the same')]));
         }
 
+        // Sync Elasticsearch values with DB if need be
+        $syncer = new QubitLftSyncer($this->resource->parentId);
+        $syncer->sync();
```

Comment:
Could pass the parent id as a parameter and do this inside the move job?

Reply:
Can we do this? Other than that it's looking great, thanks for addressing the feedback. Let me know and I'll approve.

Reply:
I've removed that as I don't think it's actually needed.